src/components/Checkbox/Checkbox.vue

```diff
@@ -19,19 +19,21 @@
     <span
       style="content: '';"
       :class="[
-        'checkmark w-4 h-4 mr-1 rounded-sm border-solid border border-gray-100 -left-5 absolute top-1',
+        'checkmark w-4 h-4 mr-1 rounded-sm border-solid border border-gray-200 -left-5 absolute top-1',
         { 'bg-white-300': disabled },
-        { '!bg-gray-100': disabled && checked },
-        { 'border-error': error },
-        { 'border-primary-500 bg-primary-500': checked }
+        { '!bg-gray-200': disabled && checked },
+        { 'border-black bg-black': checked && !disabled }
       ]"
       data-testId="checkmark"
     />
     <!-- eslint-disable-next-line vue/no-v-html -->
```

Comment:
Now that you've split the `v-html` onto a separate line this comment needs to be moved so it will disable the eslint warning properly